syntax = "proto3";

package traffic;

message TrafficRequest {
    string customer_name = 1;
    string start_date = 2;
    string end_date = 3;
    string ip = 4;
}

message TrafficResponse {
    string name = 1;
    float total_traffic = 2;
}

service TrafficService {
    rpc GetTraffic (TrafficRequest) returns (stream TrafficResponse);
}
